src/main/java/net/sf/jabref/importer/CustomImporter.java

```diff
@@ -67,6 +67,13 @@ public CustomImporter(String name, String cliId, String className, String basePa
         this.basePath = basePath;
     }
 
+    public CustomImporter(ImportFormat importer){
+        this.name = importer.getFormatName();
+        this.cliId = importer.getId();
+        this.className = importer.getClass().getName();
+        this.basePath = "src/main/java/net/sf/jabref/importer/fileformat/" +importer.getFormatName() +"Importer.java";
+    }
```

Comment:
Please call one of the other constructors instead of setting each variable here .